lib/oauther.ex

```diff
@@ -94,7 +100,7 @@ defmodule OAuther do
   end
 
   defp nonce do
-    :crypto.rand_bytes(32) |> Base.encode64
+    :crypto.rand_bytes(16) |> Hexate.encode
```

Comment:
I basically translated exactly what the Yelp ruby gem was doing to generate the nonce. I spent quite a few hours narrowing down to the nonce as the issue I was having. Sadly, the Yelp API is not RFC compliant as it definitely does not work with it base64 instead of hex encoded.